app/javascript/app/utils/ms-formatter.js

```diff
@@ -1,5 +1,5 @@
 function formatMinutes(minutes) {
-  return minutes || 0;
+  return minutes < 10 ? `0${minutes}` : minutes;
```

Comment:
It could be considered "bad-clever", but my favorite trick for this kind of behavior is:

```suggestion
  return `0${minutes}`.slice(-2);
```

**Edit:** I see this is mostly copying from the function below. Follow-up point in that case: If the logic between these functions is identical, should we have just one and rename accordingly?

```js
'01'.slice(-2) // "01"
'02'.slice(-2) // "02"
// ...
'09'.slice(-2) // "09"
'010'.slice(-2) // "10"
```